scipy/signal/wavelets.py

```diff
@@ -307,14 +307,95 @@ def ricker(points, a):
     return total
 
 
-def cwt(data, wavelet, widths):
+def morlet2(M, s, w=5):
+    """
+    Complex Morlet wavelet, designed to work with `cwt`.
+
+    Returns the complete version of morlet wavelet, normalised
+    according to `s`::
+
+        exp(1j*w*x/s) * exp(-0.5*(x/s)**2) * pi**(-0.25) * sqrt(1/s)
+
+    .. versionadded:: 1.4.0
+
+    Parameters
+    ----------
+    M : int
+        Length of the wavelet.
+    s : float
+        Width parameter of the wavelet.
+    w : float, optional
+        Omega0. Default is 5
+
+    Returns
+    -------
+    morlet : (M,) ndarray
+
+    See Also
+    --------
+    morlet : Implementation of Morlet wavelet, incompatible with `cwt`
```

Comment:
Similarly it would be good to add `morlet2` to `See Also` of `morlet`

Reply:
> 
> 
> Just a few minor remaining nitpicks.
> 
> For the release notes update, I think we need to say that `cwt` now:
> 
>     1. does time reversal, which is a backward incompatible bugfix for any time-asymmetric wavelets, and
> 
>     2. supports complex wavelets
> 
> 
> Does that sound right?

I've added this to the release notes and sorted out the other comments related to documentation.